packages/invoices/src/OutboundShipment/DetailView/modals/OutboundLineEditTable.tsx

```diff
@@ -50,7 +50,10 @@ const BatchesRow: React.FC<BatchesRowProps> = ({
         <NumericTextInput
           onChange={event => {
             const value = Math.max(Number(event.target.value), 0);
-            const newValue = Math.min(value, batch.availableNumberOfPacks);
+            const newValue = Math.min(
+              value,
+              batch.stockLine?.availableNumberOfPacks ?? 0
```

Comment:
Previously, the `DraftOutboundLine`  was extending `InvoiceLine` which had no `stockLine` or `item` fields, specifically because of the problem above - having annoying types with a bunch of required fields we didn't actually want or need.

Now, the draft type is extending `PartialOutboundLineFragment` type, which has subset of fields on `stockLine` and `item` - the ones that are needed.

So, instead of creating a `DraftOutboundLine` and adding in a bunch of fields and reassigning (i.e. putting `availableNumberOfPacks` on `DraftOutboundLine` and taking it from the `stockLine`, we can just use the `stockLine` directly